security/force_https.rst

```diff
@@ -105,3 +105,10 @@ role:
 
 It is also possible to specify using HTTPS in the routing configuration,
 see :doc:`/routing/scheme` for more details.
+
+.. note::
+
+    Forcing HTTPS while using a reverse proxy or load balancer requires a proper
+    configuration to avoid infinite redirect loops; see :doc:`/deployment/proxies`
```

Comment:
"redirection loops"?

Reply:
Chrome uses "redirect loop" in its error message:

![image](https://user-images.githubusercontent.com/202034/41536845-bc327538-72d4-11e8-98ce-fd271f8477c1.png)